x-pack/platform/plugins/private/data_visualizer/public/lite/file_status.tsx

```diff
@@ -59,29 +81,42 @@ export const FileStatus: FC<Props> = ({ fileStatus, uploadStatus, deleteFile, in
         ) : (
           <>
             <EuiFlexGroup gutterSize="s" alignItems="center">
-              <EuiFlexItem grow={8}>
-                <EuiText size="xs">
-                  <span css={{ fontWeight: 'bold' }}>{fileStatus.fileName}</span>{' '}
-                  <span>{fileStatus.fileSize}</span>
-                </EuiText>
-              </EuiFlexItem>
+              {showFileContentPreview || showFileSummary ? (
+                <EuiFlexItem grow={false}>
+                  <EuiButtonIcon
+                    iconType={expanded ? 'arrowDown' : 'arrowRight'}
+                    aria-label="expand"
+                    onClick={() => setExpanded(!expanded)}
+                  />
+                </EuiFlexItem>
+              ) : null}
+              <EuiFlexItem>
+                <EuiFlexGroup gutterSize="none" alignItems="center">
+                  <EuiFlexItem grow={8}>
+                    <EuiText size="xs">
+                      <span css={{ fontWeight: 'bold' }}>{fileStatus.fileName}</span>{' '}
```

Comment:
```suggestion
                      <span css={{ fontWeight: 'bold' }}>{fileStatus.fileName}</span>&nbsp;
```

Reply:
Updated in https://github.com/elastic/kibana/pull/213275/commits/c19ef736fe8052616736eedb6757c01c4c5df97b